core/kazoo_number_manager/src/knm_number.erl

```diff
@@ -30,6 +30,7 @@
         ,charges/2, set_charges/3
         ,to_public_json/1
         ,is_number/1
+        ,force_outbound_feature/1
```

Comment:
Is this PR about tasks cleanup? No.
It's about (see title) and this function is needed for people to be able to set this feature from a CSV job.